src/test/java/no/unit/nva/model/BookPublicationTest.java

```diff
@@ -21,6 +29,161 @@ void publicationReturnPublicationWhenInputIsValid() throws MalformedContributorE
 
         Publication publicationFromJson = objectMapper.readValue(objectMapper.writeValueAsString(document),
                 Publication.class);
-        Assertions.assertEquals(bookPublication, publicationFromJson);
+
+        assertThat(bookPublication, is(equalTo(publicationFromJson)));
+    }
+
+    @Test
+    void publicationReturnsSeriesWhenInputIsSeriesTitle() throws JsonProcessingException {
+        String book = "{\n"
```

Comment:
It seems that this jsonString was created by generating an object and then copy-pasting the serialized version of it. The problem is that such tests will fail when any other change unrelated to the test happens. 
It would be more maintainable to create a sample object, serialize it, parse it  and then verify that we have the expected types in place.  In practice, this what has been done here, but the sample object and the serialization happened once and manually.